File: app/src/main/java/tech/leonam/resultadodocertame/model/interfaces/InterfacePegaTurmas.java
package tech.leonam.resultadodocertame.model.interfaces;

import android.content.Context;

import java.util.ArrayList;

import tech.leonam.resultadodocertame.model.entidade.TurmaEntidade;

public interface InterfacePegaTurmas {
    ArrayList<TurmaEntidade> getTurmas(Context context);
}

File: app/src/main/java/tech/leonam/resultadodocertame/model/interfaces/InterfacePegaTurmas.kt
package tech.leonam.resultadodocertame.model.interfaces

import android.content.Context
import tech.leonam.resultadodocertame.modelView.entidade.TurmaEntidade

interface InterfacePegaTurmas {
    fun getTurmas(context: Context?): ArrayList<TurmaEntidade?>?
}